stackslib/src/burnchains/bitcoin/blocks.rs

```diff
@@ -437,7 +437,7 @@ impl BitcoinBlockParser {
         match (inputs_opt, outputs_opt) {
             (Some(inputs), Some(outputs)) => {
                 Some(BitcoinTransaction {
-                    txid: Txid::from_vec_be(&tx.txid().as_bytes().to_vec()).unwrap(), // this *should* panic if it fails
+                    txid: Txid::from_vec_be(tx.txid().as_bytes().as_ref()).unwrap(), // this *should* panic if it fails
```

Comment:
`as_ref()` unnecessary:

```suggestion
                    txid: Txid::from_vec_be(tx.txid().as_bytes()).unwrap(), // this *should* panic if it fails
```